api-admin/openapi.yaml

```
@@ -24,6 +24,23 @@ paths:
         500:
           description: Something goes wrong
 
+  "/user/delete":
```

Так это удаление пользователя, а не удаление пользователя из приложения.

```suggestion
  "/applications/user.remove":
```